src/packages/utils.ts

```diff
@@ -83,7 +83,7 @@ export const exportToSvg = ({
   appState = getDefaultAppState(),
   exportPadding,
   metadata,
-}: ExportOpts & {
+}: Omit<ExportOpts, "getDimensions"> & {
```

Comment:
Agreed we should omit `getDimensions` since its not getting used

Reply:
Great!